mount_sst-1m_data.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Information can be found here :
+# https://www.tecmint.com/sshfs-mount-remote-linux-filesystem-directory-using-ssh/
+#
+
+sudo sshfs -o allow_other $1@baobab.unige.ch:/sst1m/ /sst1m/
```

Comment:
I think this script should be explained in the README otherwise people might never find it ...